Makefile

```diff
@@ -304,7 +304,14 @@ create-cluster-management: $(CLUSTERCTL) ## Create a development Kubernetes clus
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
+	# Wait for cert-manager pods to be created
 	sleep 20
```

Comment:
This can probably be removed

Reply:
this sleep is here as pods are not created instantly, kubectl wait complains that no pod found
```
# Wait for cert-manager pods to be ready.
kubectl \
	--kubeconfig=$(kind get kubeconfig-path --name="clusterapi") \
	wait --for=condition=Ready --namespace=cert-manager --timeout=15m pods --all
error: no matching resources found
```